crates/orchestrator/src/setup/worker_triggers/event_bridge.rs

```diff
@@ -0,0 +1,62 @@
+use aws_sdk_eventbridge::types::{InputTransformer, RuleState, Target};
+use aws_sdk_sqs::types::QueueAttributeName;
+
+use crate::queue::job_queue::{WorkerTriggerMessage, WorkerTriggerType};
+use crate::setup::SetupConfig;
+
+#[allow(unreachable_patterns)]
+pub async fn setup_event_bridge_for_trigger_type(
+    worker_trigger_type: WorkerTriggerType,
+    config: &SetupConfig,
+    rule_name: &str,
+    worker_trigger_queue_name: &str,
+) -> color_eyre::Result<()> {
+    let config = match config {
+        SetupConfig::AWS(config) => config,
+        _ => panic!("Unsupported Event Bridge configuration"),
+    };
+    let event_bridge_client = aws_sdk_eventbridge::Client::new(config);
+    let sqs_client = aws_sdk_sqs::Client::new(config);
+
+    event_bridge_client
```

Comment:
create_cron(rule_name , cron_time: Duration, target_queue_name: String, message: String)

- `rate(1 minute)` should be created inside the event bridge implementation
- queue arn can be fetched inside create_cron
- message string will be built in setup and sent to create cron
- target_queue_name will be sent by setup to create cron
- cron_time will be hardcoded in setup and sent to create_cron